data/pop-upgrade.sh

```diff
@@ -239,29 +184,41 @@ attempt_upgrade () {
     if (upgrade || attempt_repair); then
         rm -rf  /system-update "$1"
 
+        # Set cosmic-greeter as the default display manager
+        rm -f /etc/systemd/system/display-manager.service
```

Comment:
I tried doing a 20.04 to 22.04 upgrade, since this is merging into `master` and there are no `master_*` branches in this repo.

GDM was not uninstalled, but the system booted to a TTY after the upgrade because this line removed GDM as the default. This part needs to be gated behind a version check, too (or we need to branch off `master_focal` before merging this).

Reply:
As a third alternative to the version check or separate branches, this block could alternatively check if COSMIC Greeter's installed before making it the default. If a version check's simpler, though, it should give the same end result as of right now.